stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -384,7 +384,7 @@ internal interface ChatApi {
     fun translate(messageId: String, language: String): Call<Message>
 
     @CheckResult
-    fun getSyncHistory(channelIds: List<String>, lastSyncAt: Date): Call<List<ChatEvent>>
+    fun getSyncHistory(channelIds: List<String>, lastSyncAt: String): Call<List<ChatEvent>>
```

Comment:
This seems to be a breaking change in public API.
Can we add an old function version with the `Date` param as well?

Reply:
This API is not public. There's no problem in keeping both methods, but this is conflicting with @JcMinarro's comment. Let's decide which one we prefer.

Reply:
My bad, cool then 👍